onnxruntime/test/python/quantization/test_op_gemm.py

```diff
@@ -103,6 +108,26 @@ def make_gemm(input_name, weight_shape, weight_name, bias_shape, bias_name, outp
 
         onnx.save(model, output_model_path)
 
+    @staticmethod
```

Comment:
Is there a reason staticmethod is needed? Would it be possible to lift it as a module level function to avoid staticmethod?

Reply:
What about moving it into QuantType?

Reply:
I like that idea

Reply:
done